build.gradle

```diff
@@ -254,17 +270,41 @@ dependencies {
   testCompile 'info.solidsoft.gradle.pitest:gradle-pitest-plugin:1.3.0'
   testCompile 'org.codehaus.sonar-plugins:sonar-pitest-plugin:0.5'
 
-  testCompile group: 'org.springframework.boot', name: 'spring-boot-starter-test'
+  testCompile (group: 'org.springframework.boot', name: 'spring-boot-starter-test', version: versions.springBoot) {
+    exclude group: 'junit', module: 'junit'
+  }
+  
+  testCompile files('lib/befta-fw-1.0.9-SNAPSHOT.jar')
 
+  //Added for functional test automation
+  compile group: 'org.springframework.cloud', name: 'spring-cloud-starter-openfeign', version: '2.2.1.RELEASE'
+  compile (group: 'uk.gov.hmcts.reform.auth', name: 'auth-checker-lib', version: '2.1.3') {
+    exclude group: 'uk.gov.hmcts.reform', module: 'java-logging-spring'
+  }
+  compile group: 'uk.gov.hmcts.reform', name: 'service-auth-provider-client', version: '2.0.0'
+  testCompile group: 'io.rest-assured', name: 'rest-assured', version: '3.1.0'
+  compile group: 'io.cucumber', name: 'cucumber-java', version: '5.0.0'
+  testCompile group: 'io.cucumber', name: 'cucumber-junit', version: '5.0.0'
+  testCompile 'io.github.openfeign:feign-jackson:9.7.0'
+
+  smokeTestCompile group: 'net.serenity-bdd', name: 'serenity-core', version: versions.serenity
+  smokeTestCompile group: 'net.serenity-bdd', name: 'serenity-junit', version: versions.serenity
+  smokeTestCompile group: 'net.serenity-bdd', name: 'serenity-rest-assured', version: versions.serenity
+  smokeTestCompile group: 'net.serenity-bdd', name: 'serenity-spring', version: versions.serenity
+  smokeTestCompile group: 'io.rest-assured', name: 'rest-assured', version: '3.3.0'
```

Comment:
Can you please replace the version numbers with variables wherever possible.

Reply:
Updated:
  compile group: 'io.cucumber', name: 'cucumber-java', version: versions.cucumber
  testCompile group: 'io.rest-assured', name: 'rest-assured', version: versions.rest_assured
  testCompile group: 'io.cucumber', name: 'cucumber-junit', version: versions.cucumber
  testCompile group: 'io.github.openfeign', name: 'feign-jackson', version: versions.feign_jackson

  smokeTestCompile group: 'net.serenity-bdd', name: 'serenity-core', version: versions.serenity
  smokeTestCompile group: 'net.serenity-bdd', name: 'serenity-junit', version: versions.serenity
  smokeTestCompile group: 'net.serenity-bdd', name: 'serenity-rest-assured', version: versions.serenity
  smokeTestCompile group: 'net.serenity-bdd', name: 'serenity-spring', version: versions.serenity
  smokeTestCompile group: 'io.rest-assured', name: 'rest-assured', version: versions.rest_assured